src/main/kotlin/com/github/rafaelldi/tyeplugin/api/TyeApiClient.kt

```diff
@@ -14,7 +14,10 @@ import io.ktor.client.request.get
 class TyeApiClient : Disposable {
     private val httpClient: HttpClient = HttpClient(CIO) {
         install(JsonFeature) {
-            serializer = KotlinxSerializer()
+            serializer = KotlinxSerializer(Json {
+               isLenient = true
```

Comment:
I think this option is unnecessary for this bug https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/json.md#lenient-parsing